samples/Autorotating-samples/bonus-marking-grouping/template.json

```diff
@@ -29,9 +28,13 @@
       }
     },
     {
-      "name": "AutoAlign",
-      "options": {
-        "referenceImage": "reference.jpg"
+      "name":"AutoRotate",
+      "options":{
+        "referenceImage":"reference.jpg",
+        "threshold":{
+          "value":1000000000,
```

Comment:
Need some documentation on using the value here. Can you add a Readme.md in this sample folder explaining how to decide on this value?

Reply:
This value was randomly put to check if passthrough is working ill remove it.

Reply:
this needs discussion